addons/website/static/src/components/dialog/seo.xml

```diff
@@ -169,8 +169,12 @@
         size="size"
         primaryClick="() => this.save()"
         primaryTitle="saveButton"
+        showFooter="canEditSeo"
         contentClass="contentClass">
-        <TitleDescription canEditDescription="canEditDescription"
+        <div t-if="!canEditSeo">
```

Comment:
Maybe put this t-if on the whole WebsiteDialog to not have to repeat it everywhere? + It would avoid an useless div?

Reply:
In this case, the click on menu will do nothing if you have canEditSeo to false :/ 

But field could be in RO mode and the preview replaced by the warning ?

Reply:
@rdeodoo @JKE-be I was suggesting that instead of 
```xml
<WebsiteDialog many-options-which-are-useless-in-case-of-no-rights>
     <div t-if="!can">...</div>
     <A t-if="can"/>
     <B t-if="can"/>
     <C t-if="can"/>
     <D t-if="can"/>
</WebsiteDialog>
```
You could have add:
```xml
<WebsiteDialog t-if="can" many-options>
     <A/>
     <B/>
     <C/>
     <D/>
</WebsiteDialog>
<WebsiteDialog t-else="" some-options>
    ...
</WebsiteDialog>
```

Details :)

Reply:
Note that with the latest changes, it's not the case anymore, as we kept showing the title and description but just hide the keywords and images

Reply:
That's why I said:
> Details I guess, and since you changed the design (alert + visible but disabled inputs, it makes sense as it is

:wink: